shared/wallets/common/wallet-popup.js

```diff
@@ -14,36 +14,38 @@ type WalletModalProps = {|
   // Buttons to be placed in the bottom Button Bar.
   // If none are included, the bar is not rendered.
   bottomButtons?: Array<React.Node>,
+  buttonBarDirection?: 'column' | 'row',
 |}
 
 const WalletPopup = (props: WalletModalProps) => (
   <Kb.Box2 direction="vertical" style={styles.outerContainer}>
-    {props.onBack && <Kb.HeaderHocHeader onBack={props.onBack} headerStyle={styles.header} />}
-    <Kb.Box2
-      direction="vertical"
-      fullHeight={true}
-      fullWidth={true}
-      centerChildren={true}
-      style={Styles.collapseStyles([
-        styles.container,
-        props.onBack ? {paddingTop: Styles.globalMargins.small} : {},
-        props.containerStyle,
-      ])}
-    >
-      {props.children}
-      {props.bottomButtons &&
-        props.bottomButtons.length > 0 && (
-          <Kb.Box2 direction="vertical" style={styles.buttonBarContainer} fullWidth={true}>
-            <Kb.ButtonBar
-              direction={Styles.isMobile ? 'column' : 'row'}
-              fullWidth={Styles.isMobile}
-              style={styles.buttonBar}
-            >
-              {props.bottomButtons}
-            </Kb.ButtonBar>
-          </Kb.Box2>
-        )}
-    </Kb.Box2>
+    <Kb.ScrollView style={styles.scrollView} contentContainerStyle={styles.scrollViewContentContainer}>
```

Comment:
wrap in scrollview and kill extra headerhoc header (this is already wrapped below!)